scripts/create_test_subset.py

```diff
@@ -862,10 +937,22 @@ def copy_files_in_dict(d, dataset: str, sid_replacement: tuple[str, str] = None)
         if sid_replacement is not None:
             new_path = new_path.replace(sid_replacement[0], sid_replacement[1])
 
+        job = None
+
         if not file_exists(new_path):
-            cmd = ['gcloud', 'storage', 'cp', old_path, new_path]
-            logger.info(f'Copying file in metadata: {" ".join(cmd)}')
-            subprocess.run(cmd, check=True)
+            if update_embedded_ids and sid_replacement and file_reheaderable(new_path):
+                logger.info(f'Adding job to rewrite {old_path} to {new_path}')
+                batch = get_batch()
+                job = batch.new_job(f'Reheader {new_path} from {old_path}')
+                copy_common_env(job)
```

Comment:
This isn't strictly necessary - cpg-utils copies the ENV for every job in every batch
https://github.com/populationgenomics/cpg-utils/blob/main/cpg_utils/hail_batch.py#L246